### ISSUE TITLE
Debian 12: sysrepo-plugins are not shipped in debian package?

### ISSUE THREAD

**Issue by guillaume-pais-siemens** [sysrepo/sysrepo#2750]
We are trying to install sysrepo and netopeer2 on debian 12. 
```
curl -O http://ftp.debian.org/debian/pool/main/s/sysrepo/libsysrepo6_2.0.53-6_amd64.deb
curl -O http://ftp.debian.org/debian/pool/main/s/sysrepo/sysrepo-plugind_2.0.53-6_amd64.deb
sudo apt install ./libsysrepo6_2.0.53-6_amd64.deb
sudo apt install ./sysrepo-plugind_2.0.53-6_amd64.deb
```
It seems that the plugins are not set correctly:
```
$ systemctl status sysrepo-plugind
× sysrepo-plugind.service - YANG-based configuration and operational state data>
     Loaded: loaded (/lib/systemd/system/sysrepo-plugind.service; enabled; vend>
     Active: failed (Result: exit-code) since Thu 2022-03-10 09:39:33 EST; 3s a>
    Process: 1640 ExecStart=/usr/bin/sysrepo-plugind -d -v 2 (code=exited, stat>
   Main PID: 1640 (code=exited, status=1/FAILURE)
        CPU: 21ms
```
```
$ sudo journalctl -u sysrepo-plugind
Mar 10 08:05:10 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:05:10 debian sysrepo-plugind[2104]: sysrepo-plugind error: Creating plugins dir "/usr/lib/x86_64-linux-gnu/sysrepo-plugind/plugins" failed (Read-only file system).
Mar 10 08:05:10 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=exited, status=1/FAILURE
Mar 10 08:05:10 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 08:05:10 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 1.
Mar 10 08:05:10 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:05:10 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:05:11 debian sysrepo-plugind[2115]: sysrepo-plugind error: Creating plugins dir "/usr/lib/x86_64-linux-gnu/sysrepo-plugind/plugins" failed (Read-only file system).
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=exited, status=1/FAILURE
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 2.
Mar 10 08:05:11 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:05:11 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:05:11 debian sysrepo-plugind[2127]: sysrepo-plugind error: Creating plugins dir "/usr/lib/x86_64-linux-gnu/sysrepo-plugind/plugins" failed (Read-only file system).
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=exited, status=1/FAILURE
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 3.
Mar 10 08:05:11 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:05:11 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:05:11 debian sysrepo-plugind[2132]: sysrepo-plugind error: Creating plugins dir "/usr/lib/x86_64-linux-gnu/sysrepo-plugind/plugins" failed (Read-only file system).
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=exited, status=1/FAILURE
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 4.
Mar 10 08:05:11 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:05:11 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:05:11 debian sysrepo-plugind[2136]: sysrepo-plugind error: Creating plugins dir "/usr/lib/x86_64-linux-gnu/sysrepo-plugind/plugins" failed (Read-only file system).
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=exited, status=1/FAILURE
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 5.
Mar 10 08:05:11 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Start request repeated too quickly.
Mar 10 08:05:11 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 08:05:11 debian systemd[1]: Failed to start YANG-based configuration and operational state data store.
Mar 10 08:16:43 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:26:25 debian sysrepo-plugind[2633]: [ERR] unlink() failed (No such file or directory).
Mar 10 08:26:25 debian systemd[1]: Stopping YANG-based configuration and operational state data store...
Mar 10 08:26:25 debian systemd[1]: sysrepo-plugind.service: Deactivated successfully.
Mar 10 08:26:25 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
-- Boot 2d78cbb7c74a471dac21f116f51edf0e --
Mar 10 08:26:52 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:26:53 debian sysrepo-plugind[660]: sysrepo-plugind: ./src/plugins/ds_lyb.c:162: srpds_lyb_init: Assertion `!srlyb_file_exists(srpds_name, path)' failed.
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=killed, status=6/ABRT
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 1.
Mar 10 08:26:53 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:26:53 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:26:53 debian sysrepo-plugind[700]: [WRN] Recovered a lock with a dead owner (sr_lydmods_conn_ctx_update).
Mar 10 08:26:53 debian sysrepo-plugind[700]: sysrepo-plugind: ./src/plugins/ds_lyb.c:162: srpds_lyb_init: Assertion `!srlyb_file_exists(srpds_name, path)' failed.
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=killed, status=6/ABRT
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 2.
Mar 10 08:26:53 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:26:53 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:26:53 debian sysrepo-plugind[746]: [WRN] Recovered a lock with a dead owner (sr_lydmods_conn_ctx_update).
Mar 10 08:26:53 debian sysrepo-plugind[746]: sysrepo-plugind: ./src/plugins/ds_lyb.c:162: srpds_lyb_init: Assertion `!srlyb_file_exists(srpds_name, path)' failed.
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=killed, status=6/ABRT
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 3.
Mar 10 08:26:53 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:26:53 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:26:53 debian sysrepo-plugind[809]: [WRN] Recovered a lock with a dead owner (sr_lydmods_conn_ctx_update).
Mar 10 08:26:53 debian sysrepo-plugind[809]: sysrepo-plugind: ./src/plugins/ds_lyb.c:162: srpds_lyb_init: Assertion `!srlyb_file_exists(srpds_name, path)' failed.
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=killed, status=6/ABRT
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 4.
Mar 10 08:26:53 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:26:53 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:26:53 debian sysrepo-plugind[824]: [WRN] Recovered a lock with a dead owner (sr_lydmods_conn_ctx_update).
Mar 10 08:26:53 debian sysrepo-plugind[824]: sysrepo-plugind: ./src/plugins/ds_lyb.c:162: srpds_lyb_init: Assertion `!srlyb_file_exists(srpds_name, path)' failed.
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=killed, status=6/ABRT
Mar 10 08:26:53 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:26:54 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 5.
Mar 10 08:26:54 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:26:54 debian systemd[1]: sysrepo-plugind.service: Start request repeated too quickly.
Mar 10 08:26:54 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:26:54 debian systemd[1]: Failed to start YANG-based configuration and operational state data store.
Mar 10 08:32:47 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:32:47 debian sysrepo-plugind[1354]: [WRN] Recovered a lock with a dead owner (sr_lydmods_conn_ctx_update).
Mar 10 08:32:47 debian sysrepo-plugind[1354]: sysrepo-plugind: ./src/plugins/ds_lyb.c:162: srpds_lyb_init: Assertion `!srlyb_file_exists(srpds_name, path)' failed.
Mar 10 08:32:47 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=killed, status=6/ABRT
Mar 10 08:32:47 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:32:47 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 1.
Mar 10 08:32:47 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:32:47 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:32:47 debian sysrepo-plugind[1360]: [WRN] Recovered a lock with a dead owner (sr_lydmods_conn_ctx_update).
Mar 10 08:32:47 debian sysrepo-plugind[1360]: sysrepo-plugind: ./src/plugins/ds_lyb.c:162: srpds_lyb_init: Assertion `!srlyb_file_exists(srpds_name, path)' failed.
Mar 10 08:32:47 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=killed, status=6/ABRT
Mar 10 08:32:47 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 2.
Mar 10 08:32:48 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:32:48 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:32:48 debian sysrepo-plugind[1366]: [WRN] Recovered a lock with a dead owner (sr_lydmods_conn_ctx_update).
Mar 10 08:32:48 debian sysrepo-plugind[1366]: sysrepo-plugind: ./src/plugins/ds_lyb.c:162: srpds_lyb_init: Assertion `!srlyb_file_exists(srpds_name, path)' failed.
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=killed, status=6/ABRT
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 3.
Mar 10 08:32:48 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:32:48 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:32:48 debian sysrepo-plugind[1370]: [WRN] Recovered a lock with a dead owner (sr_lydmods_conn_ctx_update).
Mar 10 08:32:48 debian sysrepo-plugind[1370]: sysrepo-plugind: ./src/plugins/ds_lyb.c:162: srpds_lyb_init: Assertion `!srlyb_file_exists(srpds_name, path)' failed.
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=killed, status=6/ABRT
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 4.
Mar 10 08:32:48 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:32:48 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 08:32:48 debian sysrepo-plugind[1373]: [WRN] Recovered a lock with a dead owner (sr_lydmods_conn_ctx_update).
Mar 10 08:32:48 debian sysrepo-plugind[1373]: sysrepo-plugind: ./src/plugins/ds_lyb.c:162: srpds_lyb_init: Assertion `!srlyb_file_exists(srpds_name, path)' failed.
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=killed, status=6/ABRT
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 5.
Mar 10 08:32:48 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Start request repeated too quickly.
Mar 10 08:32:48 debian systemd[1]: sysrepo-plugind.service: Failed with result 'signal'.
Mar 10 08:32:48 debian systemd[1]: Failed to start YANG-based configuration and operational state data store.
-- Boot 3681a354cef445f58d1f0c1ef9ce17ee --
Mar 10 09:39:31 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 09:39:31 debian sysrepo-plugind[1609]: sysrepo-plugind error: Creating plugins dir "/usr/lib/x86_64-linux-gnu/sysrepo-plugind/plugins" failed (Read-only file system).
Mar 10 09:39:31 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=exited, status=1/FAILURE
Mar 10 09:39:31 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 1.
Mar 10 09:39:32 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 09:39:32 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 09:39:32 debian sysrepo-plugind[1617]: sysrepo-plugind error: Creating plugins dir "/usr/lib/x86_64-linux-gnu/sysrepo-plugind/plugins" failed (Read-only file system).
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=exited, status=1/FAILURE
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 2.
Mar 10 09:39:32 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 09:39:32 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 09:39:32 debian sysrepo-plugind[1631]: sysrepo-plugind error: Creating plugins dir "/usr/lib/x86_64-linux-gnu/sysrepo-plugind/plugins" failed (Read-only file system).
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=exited, status=1/FAILURE
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 3.
Mar 10 09:39:32 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 09:39:32 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 09:39:32 debian sysrepo-plugind[1636]: sysrepo-plugind error: Creating plugins dir "/usr/lib/x86_64-linux-gnu/sysrepo-plugind/plugins" failed (Read-only file system).
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=exited, status=1/FAILURE
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 4.
Mar 10 09:39:32 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 09:39:32 debian systemd[1]: Started YANG-based configuration and operational state data store.
Mar 10 09:39:32 debian sysrepo-plugind[1640]: sysrepo-plugind error: Creating plugins dir "/usr/lib/x86_64-linux-gnu/sysrepo-plugind/plugins" failed (Read-only file system).
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Main process exited, code=exited, status=1/FAILURE
Mar 10 09:39:32 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 09:39:33 debian systemd[1]: sysrepo-plugind.service: Scheduled restart job, restart counter is at 5.
Mar 10 09:39:33 debian systemd[1]: Stopped YANG-based configuration and operational state data store.
Mar 10 09:39:33 debian systemd[1]: sysrepo-plugind.service: Start request repeated too quickly.
Mar 10 09:39:33 debian systemd[1]: sysrepo-plugind.service: Failed with result 'exit-code'.
Mar 10 09:39:33 debian systemd[1]: Failed to start YANG-based configuration and operational state data store.
```

**Comment by michalvasko** [comment]
These packages were in the process of being finished, but are halted currently. However, I honestly do not know what to do about
```
Mar 10 08:05:10 debian sysrepo-plugind[2104]: sysrepo-plugind error: Creating plugins dir "/usr/lib/x86_64-linux-gnu/sysrepo-plugind/plugins" failed (Read-only file system).
```
Why would the libraries be mounted on read-only filesystem?

**Comment by guillaume-pais-siemens** [comment]
We also don t understand. The filesystem is not read-only.

**Comment by guillaume-pais-siemens** [comment]
The problem is solved in version [v2.1.42](https://github.com/sysrepo/sysrepo/releases/tag/v2.1.42)
